src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Document/Document.ts

```diff
@@ -324,7 +324,7 @@ export var postEdit = (
     }
 
     var commit = () => {
-        return adhHttp.deepPost(_.flatten<ResourcesBase.Resource>([documentVersion, paragraphItems, paragraphVersions]))
+        return adhHttp.deepPost(_.flatten<ResourcesBase.IResource>([documentVersion, paragraphItems, paragraphVersions]))
```

Comment:
`ResourcesBase.Resource` is a class with functionality, `ResourcesBase.IResource` is only an interface. So it is "more minimal".